.travis.yml

```diff
@@ -44,7 +44,7 @@ before_install:
   - "touch .travis-release-requirements.txt"
 
 install:
-  - "sudo apt-get -qy install apache2 libapache2-mod-wsgi ssl-cert poppler-utils git subversion nodejs --fix-missing"
+  - "sudo apt-get -qy install apache2 libapache2-mod-wsgi libapache2-mod-xsendfile ssl-cert poppler-utils git subversion nodejs --fix-missing"
```

Comment:
Is it necessary to enable it (`sudo a2enmod xsendfile`) too or it's enabled by defauilt?